src/components/styles/book.css

```diff
@@ -1,5 +1,21 @@
 .divider {
   height: 0.125rem;
-  margin: 2.5rem 0.063rem 1.813rem 0;
+  margin: 20px 89px 20px 89px;
   border: solid 1px var(--border-color);
 }
+
+.main {
+  height: 100%;
+}
+
+@media (min-width: 768px) {
+  .container {
+    display: flex;
+    gap: 10px;
+  }
+
+  .scroll {
+    overflow-y: scroll;
+    height: 400px;
```

Comment:
- [ ] There is so much white space in your form as compared to the given design. I believe it is because of this height value. 400px is a bit smaller. It will be nice if you could increase it to 800px. 

Take a look at a screenshot of your project.
![a 1](https://user-images.githubusercontent.com/39523312/155830504-a9316a82-e555-42af-8a38-5aa5d156e151.png)